### ISSUE TITLE
Use similar audio formats for input/output

### ISSUE THREAD

**Issue by ltoenning** [swift-project/pilotclient#414]
With Qt5 we used the nearest format function to find a matching audio format. 
This method is no longer available with Qt6.

To support a wider range of audio formats and not only the exact one specified in code, such as feature should be re-added.

It needs to be researched if this has any side effects on other subsystems (AFV etc.)
